package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "formbuilder-BrennonDenny",
+    "version": "1.0.0",
+    "description": "",
+    "main": "app.js",
+    "scripts": {
```

Comment:
I would recommend adding some custom scripts to make the testing and running of the app simpler. (Condenses "node app" to "npm start")